Task_HelloWorld/index.js

```diff
@@ -1,26 +1,42 @@
-// Print hello World
-// Write your code here
-
-// Write your code above
-
-//Declare a variable i and set its value to 5
-// Write your code here
-
-// Write your code above
-
-//Write the if else statement to console "Number is odd" if the value of i is odd and "Number is even" if the value of i is even.
-// Write your code here
-
```

Comment:
Consider adding comments to clarify the purpose of the condition, such as '// Check if i is even'